docs/libvirt-pod-networking.md

```diff
@@ -4,7 +4,7 @@ Author: Fabian Deutsch \<fabiand@redhat.com\>
 
 ## Introduction
```

Comment:
If this is the case, then I can try and argue that the pod network has nothing to do with the virtual network of the VM. There is added value in having a separation between the "host" network and the virt network, in this case, we can consider the flat pod network (that connects all pods in the cluster) to be what we call a host network and the networks exposed to the VM/s to be a virt network.
Do we have a specific need for the flat default kubernetes network in the VM/s? I understand that we need it for managing and accessing the pods, but it is not clear to me why we need the VM/s to access it.. They could just use a different network, in parallel or "above" that one.
And if a scenario arrives that requires the VM/s to access a service that is exposed in the pods network for some reason, just use a plain router to connect the two networks.